contributing/docs-style.md

```diff
@@ -0,0 +1,379 @@
+  ```
+
+You can change the text of the label at any time by passing it as an argument, too:
+
+```liquid
+{{ "{% info Recommendation " }}%}
+```
+
+### Links
+
+Adding links to documentation is an effective way to guide the user into a better understanding of the topic being discussed without the risk of overwriting.
+
+* **Links between sections in the page.** Each heading generates an implicit identifier to directly link it within a page. This identifier is generated by making the heading lowercase and replacing internal spaces with hyphens.
+
+  * **Heading:** ## Heading Title
+  * **ID:** #heading-title
+  <p></p>
+
+  ```liquid 
+  [Link Text](#heading-title) 
+  ```
+
+* **Links between files in the repository.** The base relative path to the domain of the documentation is available as a configuration variable named `baseurl`.
+
+  ```liquid 
+  [Link Text]({{ "{{ site.baseurl " }}}}/path/to/link_page.html)
+  ```
+
+* **External links**
+
+  ```liquid 
+  [Link Text](external_url)
+  ```
+
+<div class="alert alert-warning">
+  <h3>Best Practice</h3>
+  Use descriptive links that provide information on the action or destination. For example, avoid using "Learn More" or "Click Here" links.
+</div>
+
+### Visual Elements
+
+Figures and other visual elements are placed under the root _fig_ folder and can be referenced in documentation pages using a syntax similar to that of links:
+
+```liquid 
+![Picture Text]({{ "{{ site.baseurl " }}}}/fig/image_name.png){:height="200px" width="200px"}
+```
+
+Or using plain HTML:
+
+{% highlight html %}
+<img src="{{ site.baseurl }}/fig/image_name.png" class="center" height="200px" width="200px"/>
+{% endhighlight %}
+
+<div class="alert alert-warning">
+  <h3>Best Practice</h3>
+  Use flowcharts, tables and figures where appropriate or necessary for additional clarification, but never as a standalone source of information. Make sure that any text included in those elements is large enough to read and that the overall resolution is adequate.
+</div>
+
+### Code
+
+* **Inline code.** Small code snippets or references to language constructs in normal text flow should be highlighted using surrounding backticks ( **\`** ).
+
+* **Code blocks.** Code that represents self-contained examples, feature walkthroughs, demonstration of best practices or other useful scenarios should be wrapped using a fenced code block with appropriate [syntax highlighting](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers). One way of achieving this with markup is:
+
+
+  ```liquid 
+  {{ "{% highlight java"}}%} 
+     your code
+  {{"{% endhighlight"}}%} 
+  ```
+
+
+  which is equivalent to using triple backticks ( **\`\`\`** ):
+
+
+  ```liquid
+  {{ "```java 
+     your code"
+  }}```
+  ```
+
+
+  When specifying multiple programming languages, each code block should be styled as a tab:
+
+  ```html
+  <div class="codetabs" markdown="1">
+
+  <div data-lang="java" markdown="1"> … </div>
```

Comment:
expand example and show that syntax highlighting needs to be enabled here as well:

```
<div class="codetabs" markdown="1">

  <div data-lang="java" markdown="1">
{% highlight java%} 
// Java Code
{% endhighlight%}
  </div>

  <div data-lang="scala" markdown="1">
{% highlight scala%} 
// Scala Code
{% endhighlight%}
  </div>

</div>
```